boards/riscv/niosv_g/niosv_g_defconfig

```diff
@@ -0,0 +1,12 @@
+# Copyright (C) 2023, Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_NIOSV=y
+CONFIG_SOC_NIOSV_G=y
+CONFIG_BOARD_NIOSV_G=y
+CONFIG_CONSOLE=y
+CONFIG_PRINTK=y
+CONFIG_SERIAL=y
+CONFIG_UART_CONSOLE=y
+CONFIG_UART_ALTERA_JTAG=y
+CONFIG_XIP=n
```

Comment:
is this default y by default?

Reply:
Yes. CONFIG_XIP is y by default for RISCV arch.